libcxx/docs/ReleaseNotes/19.rst

```diff
@@ -133,8 +135,8 @@ LLVM 20
 
 LLVM 21
 ~~~~~~~
-TODO
 
+- Libc++ will no longer back-port non-critical bug fixes to C++03.
```

Comment:
This should briefly describe the overall effort, why we're doing it and what it means for users. And it should have a link to the RFC for more information.